playground/AzureKusto/AzureKusto.Worker/AzureKusto.Worker.csproj

```diff
@@ -9,6 +9,9 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Kusto.Data" />
+    <PackageReference Include="Microsoft.Azure.Kusto.Ingest" />
+    <PackageReference Include="Polly.Extensions" />
+    <PackageReference Include="System.Text.Json" VersionOverride="9.0.7" /><!-- TODO: Required by Ingest; what to do here? -->
```

Comment:
Flagging this line for reviewers. The Microsoft.Azure.Kusto.Ingest library requires STJ 9.0.7, which conflicts with our version in Directory.Packages.props to use the version that comes with the runtime.

What's the right behavior here?

Reply:
Do we know why Microsoft.Azure.Kusto.Ingest is forcing `net8.0` apps to use System.Text.Json v9? That isn't typical. It shouldn't require lifting BCL libraries out to later versions.